### DISCUSSION TITLE
start-cli -e 'query' just return 1000 lines，redirect to file  can't auto stop

### DISCUSSION THREAD

**Discussion by xiaoyekanren** [apache/iotdb#3925] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7812383/132344008-ef47866f-c694-44e2-8798-548838c4c52f.png)
when i use start-cli.sh to execute a query sql ,its just return 1000 rows,

![image](https://user-images.githubusercontent.com/7812383/132344020-93fd90f5-d338-4ea6-8f8f-7607b3cc8a07.png)

and i use 
`start-cli.sh -e "sql" > redirect.txt`
its just return 1000 rows and must use ctrl+c to stop

**Comment by HTHou** [comment, marked as answer]
A known issue. 
https://issues.apache.org/jira/browse/IOTDB-1275 

One simple solution is using `start-cli.sh -maxPRC -1 -e "sql" > redirect.txt`

**Reply by xiaoyekanren** [reply]
thanks
